lago/sysprep.py

```diff
@@ -117,6 +117,14 @@ def config_net_interface_dhcp(iface, hwaddr):
     )
 
 
+def edit(filename, expression):
+    editstr = '%s:""%s""' % (filename, expression)
```

Comment:
Yes, it is. virt-edit specifically mentions that it should be used to ensure escaping.